package.json

```diff
@@ -37,36 +37,36 @@
     "watch": "echo NPM script \"watch\" has been replaced with \"start\"."
   },
   "devDependencies": {
-    "@babel/plugin-proposal-class-properties": "^7.12.1",
-    "@babel/plugin-proposal-object-rest-spread": "^7.12.1",
-    "@babel/plugin-transform-runtime": "^7.12.1",
-    "@babel/preset-env": "^7.12.1",
-    "@babel/preset-react": "^7.12.5",
-    "@babel/preset-typescript": "^7.12.1",
-    "@babel/runtime": "^7.12.5",
+    "@babel/plugin-proposal-class-properties": "^7.13.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.13.8",
+    "@babel/plugin-transform-runtime": "^7.13.10",
+    "@babel/preset-env": "^7.13.12",
+    "@babel/preset-react": "^7.13.13",
+    "@babel/preset-typescript": "^7.13.0",
+    "@babel/runtime": "^7.13.10",
     "abort-controller": "^3.0.0",
     "babel-jest": "^26.6.3",
-    "concurrently": "^5.3.0",
-    "core-js": "^3.6.5",
-    "cross-env": "^7.0.2",
+    "concurrently": "^6.0.1",
+    "core-js": "^3.10.0",
+    "cross-env": "^7.0.3",
     "dotenv": "^8.2.0",
     "error-stack-parser": "^2.0.6",
     "get-port": "^5.1.1",
-    "global-agent": "^2.1.12",
-    "husky": "^4.3.0",
+    "global-agent": "^2.2.0",
+    "husky": "^6.0.0",
     "jest": "^26.6.3",
-    "jest-image-snapshot": "^4.2.0",
+    "jest-image-snapshot": "^4.4.1",
     "jest-junit": "^12.0.0",
     "jest-trx-results-processor": "^2.2.0",
-    "lerna": "^3.22.1",
-    "lint-staged": "^10.5.1",
+    "lerna": "^4.0.0",
+    "lint-staged": "^10.5.4",
     "lolex": "^6.0.0",
     "node-fetch": "^2.6.1",
     "p-defer": "^3.0.0",
-    "prettier": "^2.1.2",
+    "prettier": "^2.2.1",
     "react": "16.8.6",
     "react-dom": "16.8.6",
-    "selenium-webdriver": "^4.0.0-alpha.7",
+    "selenium-webdriver": "^4.0.0-beta.2",
```

Comment:
Any status on changing to version 89?

Reply:
Can you leave a comment on #3826?

Reply:
Will do the Docker image in another PR, it may affect our screenshots.